frontend/src/js/ProductPage/ProductPage.jsx

```diff
@@ -26,7 +26,7 @@ class ProductPage extends React.Component {
         console.log(this.state)
       }
     }
-    xhr.open('GET', 'http://localhost:4000/getIndividualItem/${this.props.params.itemID}')
+    xhr.open('GET', `/getIndividualItem/${this.props.params.itemID}`) // eslint-disable-line
```

Comment:
This prevents the dev server from working properly. Is there another way to do this?